authors/apps/appnotifications/views.py

```diff
@@ -30,6 +30,13 @@ def update(self, request, *args, **kwargs):
         serializer.save()
         return Response(serializer.data, status=status.HTTP_200_OK)
 
+    def get(self, request, *args, **kwargs):
+        user = UserNotification.objects.get(user=request.user)
+        if user:
+            serializer = self.serializer_class(user, many=False)
+            print(serializer.data)
```

Comment:
Why are you printing inside this function?

Reply:
I have removed it, I had forgotten about it